test/services/groups/transfer_service_test.rb

```diff
@@ -79,5 +79,40 @@ def setup
       end
       assert_enqueued_with(job: UpdateMembershipsJob)
     end
+
+    test 'metadata summary updates after group transfer' do
+      # Reference group/projects descendants tree:
+      # group12 < subgroup12b (project30 > sample 33)
+      #    |
+      #    ---- < subgroup12a (project29 > sample 32) < subgroup12aa (project31 > sample34 + 35)
+      @group12 = groups(:group_twelve)
+      @subgroup12a = groups(:subgroup_twelve_a)
+      @subgroup12b = groups(:subgroup_twelve_b)
+      @subgroup12aa = groups(:subgroup_twelve_a_a)
+
```

Comment:
Same comment as above but before calling the transfer service

Reply:
Updated in f76e47740b96955688a5b0d9e0c1c418deacaad4